scripts/covid19_india/medical_tests_in_data/preprocess.py

```diff
@@ -0,0 +1,31 @@
+import json
+import csv
+import urllib.request
+
+# 1. All counts are at India level, so geographic reference code will always be IN
+# 2. Only cumulative counts are available at this point
+
+INDIA = "IN"
+
+output_columns = ['Date', 'isoCode', 'CumulativeCount_MedicalTest_COVID_19']
+
+# Get the data from github.com/datameet/covid19n which scrapes the data
+# On daily basis from the ICMR PDFs, get from master
+
+with urllib.request.urlopen(
+        "https://raw.githubusercontent.com/datameet/covid19/master/data/icmr_testing_status.json"
```

Comment:
Do we have state-level data too?

Reply:
- We are still figuring out to get that data on a regular basis.
- We do have Confirmed, Recovered, Deaths by state. In the next PR.